src/NuGet.Configuration/PackageSource/NuGetConstants.cs

```diff
@@ -0,0 +1,20 @@
+﻿namespace NuGet.Configuration
+{
+    public static class NuGetConstants
+    {
+        public static readonly string V3FeedUrl = "https://api.nuget.org/v3/index.json/";
+        public static readonly string V2FeedUrl = "https://www.nuget.org/api/v2/";
+        public static readonly string V2LegacyOfficialPackageSourceUrl = "https://nuget.org/api/v2/";
+        public static readonly string V2LegacyFeedUrl = "https://go.microsoft.com/fwlink/?LinkID=230477";
+        
+        public static readonly string V1FeedUrl = "https://go.microsoft.com/fwlink/?LinkID=206669";
+        public static readonly string DefaultGalleryServerUrl = "https://www.nuget.org";
+        public static readonly string DefaultSymbolServerUrl = "http://nuget.gw.symbolsource.org/Public/NuGet";
+        
+        public static readonly string ReadmeFileName = "readme.txt";
+        public static readonly string NuGetSolutionSettingsFolder = ".nuget";
+
+        public static readonly string V3FeedName = "nuget.org(v3)";
```

Comment:
We should have v2/v3 on both feed names. Do spaces work well in the feed names?  Could we do "nuget.org (v3)" and "nuget.org (v2)?

The other option is to only put the qualifier on the v2 feed.  Something like "nuget.org (Legacy API)".